### PR TITLE
fix: vault filter flyout button text to be reactive

### LINKED ISSUE #1350: Vault filter flyout button text for all vaults is not reactive
The button text does not get updated when we switch the language (but works after refreshing the page):
image

### PR DESCRIPTION
**Pull request by HeidiHan0000**
Closes #1350 
Credential Details Page after change:

https://user-images.githubusercontent.com/44453261/173065207-cbeb4eae-216c-4ff7-ba2b-0bb8e15f763d.mov

Signed-off-by: heidihan0000 <daeun.han@avast.com>

### CODE REVIEW

**Comment by birtony** [comment]
Looks good! Please, double check that it works when you create multiple vaults and switch between them/change language

File: cmd/wallet-web/src/translations/CredentialDetails/fr.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "allVaultLabel": "Ajouter une chambre forte",
```
Comment (HeidiHan0000) [inline cmd/wallet-web/src/translations/CredentialDetails/fr.json:2]: I changed the AllVaultLabel french translation from `Ajouter une chambre forte` -> `Toutes les chambres fortes` based on  my (limited) French and [this abstract](https://app.abstract.com/projects/9ec92568-60a2-4fee-8215-5aa997d52428/branches/dc10bcee-d9a1-44f9-b644-8f843c6f1a19/commits/68d43e7308faabf1b365be94685d85413c7ad001/files/95d5757f-f10e-43f6-ad50-0faa236217fd/layers/B18B0F73-D1C3-4BE9-849F-6EADB710BA59?collectionId=ba692773-75df-473a-a41c-38a0647809a4&collectionLayerId=595d57fd-3a25-405f-9db8-113cddcd1d67) page, please correct me if I am wrong!

(screenshot of Abstract Credentials Page: )
image

**Comment by birtony** [comment]
> > Looks good! Please, double check that it works when you create multiple vaults and switch between them/change language
> 
> 
> 
> "Default Vault" should not get translated since it is a name, right? @birtony 
> 
> 
> 
> https://user-images.githubusercontent.com/44453261/173068452-89815784-3b58-4499-9670-5c26bd02ebca.mov
> 
> 
> 
> 

Yes, this is correct 👍🏼

**Review by birtony — APPROVED** [review]

File: cmd/wallet-web/src/translations/CredentialDetails/fr.json (hunk 1 above)
Comment (talwinder50) [reply cmd/wallet-web/src/translations/CredentialDetails/fr.json:2]: Can we confirm this with ux team ?

**Review by talwinder50 — APPROVED** [review]

**Pull request merged**